charts/generic/values.yaml

```diff
@@ -222,3 +223,28 @@ additionalContainers: []
 
 # -- [Topology Spread Constraints](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/)
 topologySpreadConstraints: []
+
+hooks:
+  # -- Enable or disable all hooks
+  enabled: false
+
+  # -- Hooks to be deployed. The map key is used inJob name
+  jobs:
+    pre-upgrade:
```

Comment:
Having this default example here will mean that it is merged with user defined values if the user has no hook called "pre-upgrade". So users might be surprised by always having this dummy hook if they define other hooks.
We should comment the example out. The downside is that we have to copy the example to the CI values if we want to continue testing it.

Reply:
Good point, updated the values.